Loudspeaker_Explorer.md

```diff
@@ -2227,7 +2227,7 @@ lsx.alt.make_chart(
         .transform_calculate(VsSpeaker=alt.datum['VsSpeakerInfo']['Speaker'])
         .transform_calculate(VsValue=alt.datum['VsSpeakerInfo']['value'])
         .transform_calculate(diff=alt.datum['value'] - alt.datum['VsValue'])
-        .transform_calculate(percent='(1 - cumulativeNormal(0, datum.diff, datum.olive_standard_error)) * 100')
+        .transform_calculate(percent='(1 - cumulativeNormal(0, datum.diff, sqrt(2*datum.olive_standard_error^2))) * 100')
```

Comment:
Nice catch. Looking at your explanation, it seems like I was indeed wrong [in my original post](https://www.audiosciencereview.com/forum/index.php?threads/master-preference-ratings-for-loudspeakers.11091/page-23#post-429503) when I wrote that the sum of two normally distributed random variables has the same standard deviation as both variables.